.github/workflows/ci.yml

```diff
@@ -596,6 +596,71 @@ jobs:
         run: |
           ./uv pip install anyio
 
+  integration-test-pypy-windows:
+    needs: build-binary-windows
+    name: "integration test | pypy on windows"
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "pypy3.9"
+
+      - name: "Download binary"
+        uses: actions/download-artifact@v4
+        with:
+          name: uv-windows-${{ github.sha }}
+
+      - name: PyPy info
+        run: Get-Command pypy
+
+      - name: "Create a virtual environment"
+        run: |
+          $pypy = (Get-Command pypy).source
+          .\uv.exe venv -p $pypy
+
+      - name: "Check for executables"
+        shell: python
+        run: |
+          import sys
+          from pathlib import Path
+
+          def binary_exist(binary):
+            binaries_path = Path(".venv\\Scripts")
+            if (binaries_path / binary).exists():
+              return True
+            print(f"Executable '{binary}' not found in folder '{binaries_path}'.")
+
+          all_found = True
+          expected_binaries = [
+              "pypy3.9.exe",
+              "pypy3.9w.exe",
+              "pypy3.exe",
+              "pypyw.exe",
+              "python.exe",
+              "python3.9.exe",
+              "python3.exe",
+              "pythonw.exe",
+          ]
+          for binary in expected_binaries:
+            if not binary_exist(binary):
+              all_found = False
+
+          if not all_found:
+            print("One or more expected executables were not found.")
+            sys.exit(1)
+
+      - name: "Check version"
+        run: |
+          & .venv\Scripts\python --version
```

Comment:
Could you add a couple of these that invoke the non- "python" copies of the executable (and for Linux too)